crates/viewer/re_space_view_dataframe/src/latest_at_table.rs

```diff
@@ -19,6 +21,10 @@ pub(crate) fn latest_at_table_ui(
 ) {
     re_tracing::profile_function!();
 
+    //
+    // DATA
+    //
```

Comment:
nit: if the separation is important to you, why don't you make it several functions? :)

Reply:
Short answer, because of the tight coupling these functions would have, implying tuple return values and loads of parameters. I feel this would be strictly worse for such cases (aka. a "mono-function" file with that very "sequential" nature to it).

I'm not super happy with the status quo either. I wish rustfmt would let me insert an additional newline or two between "sections". Could go with the "// ================"-style comment, but I'm more used to have them _between_ functions.

Arf.. life is though.

Reply:
Gonna merge as is. This code is likely to be rewritten 3 times in the next couple of months anyways :)